packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -306,7 +306,7 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
       return;
     }
     return (
-      <div class="flex justify-between text-sm">
```

Comment:
Those similar changes are because I added the text-size as part of the CSS variables